docker-compose.yml

```diff
@@ -54,9 +54,12 @@ x-sentry-defaults: &sentry_defaults
 x-snuba-defaults: &snuba_defaults
   <<: *restart_policy
   depends_on:
-    - redis
-    - clickhouse
-    - kafka
+    clickhouse:
+      condition: service_healthy
```

Comment:
Having to state this explicitly is crazy! We should expand this to `sentry-defaults` too.

Reply:
Took me some time to figure this out. Glad that I could help.